doc/long-help.txt

```diff
@@ -155,6 +155,17 @@ Options:
           escape sequences unless the syntax highlighting language is plain text. Possible values:
           auto, always, *never*.
 
+      --disable-line-limits
```

Comment:
The fewer options the better IMO. So I don't think we need this one, since I don't think we need two settings (see comment below).